scripts/run.sh

```diff
@@ -119,6 +119,39 @@ TAG=${DETECTOR_VERSION:-main}/${DETECTOR_CONFIG}/${TAG_PREFIX:+${TAG_PREFIX}/}${
 if [[ "$EXTENSION" == "hepmc3.tree.root" ]]; then
   # Define location on xrootd from where to stream input file from
   INPUT_FILE=${XRDRURL}/${XRDRBASE}/${INPUT_FILE}
+
+  # Mix background events
+  {
+    date
+    eic-info
+    prmon \
+      --filename ${LOG_TEMP}/${TASKNAME}.hepmcmerger.prmon.txt \
+      --json-summary ${LOG_TEMP}/${TASKNAME}.hepmcmerger.prmon.json \
```

Comment:
You're using LOG_TEMP before defining it.

Reply:
I think it would make sense to keep the job setup before the merging. Just add another if there.

Reply:
alright. moved it back to where it was with a if clause